ros_ws/src/interpreter_pkg/interpreter_pkg/interpreter.py

```diff
@@ -0,0 +1,168 @@
+import os
+import rclpy
+from rclpy.node import Node
+from example_interfaces.msg import String
+
+import langchain
+from langchain.chains import LLMChain
+from langchain.chat_models import ChatOpenAI
+from langchain.llms import OpenAI
+from langchain.prompts import (ChatPromptTemplate, HumanMessagePromptTemplate,
+                                SystemMessagePromptTemplate)
+from langchain.schema import  HumanMessage, SystemMessage
+
+from ament_index_python.packages import get_package_share_directory
+
+data_dir = os.path.join(get_package_share_directory("interpreter_pkg"))
+OpenAI.api_key=os.environ["OPENAI_API_KEY"]
+MODEL_TO_USE="gpt-4"
```

Comment:
Should be easily configureable

Reply:
I'm going to go back to 3.5 turbo to check.  That seemed to work better. Or are you saying it should be a parameter passed at runtime?  I set it up as a constant since I palnned to use the same model for the 3 different calls

Reply:
Also you should try 4-turbo versus just straight 4 as its 1/10th the price and faster.